### PR TITLE
Update Merlin Examples to be compatible with latest Merlin

### PR DESCRIPTION
**Pull request by karzuo**
**What this PR does / why we need it**:
Context:
Merlin examples are created some time back. Over time, Merlin has continued to evolve and some of the packages used may have become incompatible. In this MR, we updated all examples such that users only need to change the relevant configs in the notebooks..

**Checklist**

- [ ] Added unit test, integration, and/or e2e tests
- [x] Tested locally
- [x] Updated documentation
- [ ] Update Swagger spec if the PR introduce API changes
- [ ] Regenerated Golang and Python client if the PR introduce API changes

### CODE REVIEW

File: examples/transformer/custom-transformer/PyFunc-Transformer.ipynb [hunk 1]
```diff
@@ -1,723 +0,0 @@
-{
```
[the remark this answers is not in the document]
Comment (karzuo) [reply examples/transformer/custom-transformer/PyFunc-Transformer.ipynb:1]: it's not needed. the text says PyFunc, but the code is Pytorch which is the same as the other notebook in same folder. This should be a draft forgotten to be deleted.

[the remark this answers is not in the document]
Comment (ariefrahmansyah) [reply examples/transformer/custom-transformer/PyFunc-Transformer.ipynb:1]: Both of them are different and none is draft. The Pytorch Transformer notebook is for standard model deployment while the Pyfunc Transformer notebook is pyfunc model that wraps pytorch model.

File: examples/transformer/custom-transformer/requirements.txt [hunk 2]
```diff
@@ -1,2 +1,4 @@
-merlin-sdk==0.10.0.dev1
-pandas
\ No newline at end of file
+pandas
```
Comment (ariefrahmansyah) [inline examples/transformer/custom-transformer/requirements.txt:1]: why merlin-sdk is removed? should just remove the version

File: examples/transformer/standard-transformer/requirements.txt [hunk 3]
```diff
@@ -1,2 +1,3 @@
 merlin-sdk==0.12.2
```
Comment (ariefrahmansyah) [inline examples/transformer/standard-transformer/requirements.txt:1]: one more pinned merlin-sdk 😅

**Review by ariefrahmansyah — APPROVED** [review]
lgtm!

**Pull request merged**